app/controllers/Application.scala

```diff
@@ -146,3 +150,9 @@ class Application(
       Redirect("/" + path, request.queryString, MOVED_PERMANENTLY)
   }
 }
+
+case class VisitToken(value: String) extends AnyVal
+object VisitToken {
+  def apply(seed: LocalDateTime): VisitToken =
+    new VisitToken(s"${seed.toEpochSecond(ZoneOffset.UTC)}")
```

Comment:
this only changes once a second, meaning it may be predictable.  Should we be using a secure/unpredictable random number?

Reply:
does this mean two pages served within the same second would have the same `VisitToken`?

Reply:
I have changed to use SecureRandom.  it now enters a long as hex into the page e.g.
`window.guardian.visitToken = "3ada0bd900da76d3";`
 